### ISSUE TITLE
A/P Alarm

### ISSUE THREAD

**Issue by DeputyLOL** [Alexustas/Props-requests#8]
Do you think you could make a button/alarm? It would look like this:
![image](https://cloud.githubusercontent.com/assets/12616126/10334826/b6ccdc6c-6ce4-11e5-90fc-b09488641d83.png)
The lights are square and have a transparent section around the edges. (you can ignore the FMC one)
Do you think you could also do the back plate as well.

**Comment by Alexustas** [comment]
do you need a  "TEST" section on this panel?

**Comment by DeputyLOL** [comment]
no thanks

**Comment by DeputyLOL** [comment]
New note. The A/T and A/P button must be able to flash red or amber.

**Comment by DeputyLOL** [comment]
I don't think you need to bother with this, I found another solution. Thanks anyway.
You can close this now if you want, ill leave it up to you.
